backend/src/main/resources/application.properties

```diff
@@ -1,11 +1,11 @@
-spring.profiles.active=dev
+spring.profiles.active=@spring.profiles.active@
 
 # REST related config
 server.servlet.context-path = /api/v1
 
 server.port=8080
 
-spring.data.neo4j.uri=bolt://localhost:7687
+spring.data.neo4j.uri=bolt://dev.money-tree.tech:7687
```

Comment:
Here we could use the container`s name since we`re using docker compose. But the dev endpoinr will still work.